src/App.tsx

```diff
@@ -1,26 +1,255 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
 
-const USER_ID = 0;
+import { UserWarning } from './UserWarning';
+import {
+  USER_ID,
+  getTodos,
+  postTodos,
+  deleteTodos,
+  patchTodo,
+} from './api/todos';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { FilterStatus } from './types/FilterStatus';
+import { getFilteredTodos } from './utils/getFilteredTodos';
+import { ErrorNotification } from './Components/ErrorNotification';
+import { getRandomNumberId } from './utils/getRandomNumberId';
+import { getCountOfCompletedTodos } from './utils/getCountOfCompletedTodos';
+import { ErrorMessage } from './types/ErrorMessage';
 
 export const App: React.FC = () => {
+  const [newTitle, setNewTitle] = useState<string>('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterStatus>(FilterStatus.All);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessage | string>('');
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [todoIdsInLoading, setTodoIdsInLoading] = useState<number[]>([]);
+  const [todoEditingId, setTodoEditingId] = useState<number | null>(null);
```

Comment:
We can add blank lines, it will improve readability
```suggestion

  const [todos, setTodos] = useState<Todo[]>([]);
  
  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
  const [filterBy, setFilterBy] = useState<FilterStatus>(FilterStatus.All);
  
  const [isLoading, setIsLoading] = useState<boolean>(false);
  const [todoIdsInLoading, setTodoIdsInLoading] = useState<number[]>([]);
  const [errorMessage, setErrorMessage] = useState<ErrorMessage | string>('');
  
  const [newTitle, setNewTitle] = useState<string>('');
  const [todoEditingId, setTodoEditingId] = useState<number | null>(null);
```